app/assets/javascripts/discourse/app/templates/preferences/profile.hbs

```diff
@@ -1,3 +1,5 @@
+<PluginOutlet @name="before-user-preferences-profile" @tagName="span" @connectorTagName="div" @args={{hash model=this.model}} />
```

Comment:
Looking at other outlets, our convention tends to be having `-before` as a suffix. Also, the `@tagName` and `@connectorTagName` arguments are considered deprecated, so let's drop them here:
```suggestion
<PluginOutlet @name="user-preferences-profile-before" @args={{hash model=this.model}} />
```